lua/iswap.lua

```diff
@@ -8,17 +8,6 @@ local err = util.err
 
 local M = {}
 
-M.config = default_config
```

Comment:
I'm curious why you moved these definitions without seemingly changing anything other than adding the `init()` call

Reply:
Just moved the init function up here to have it declared before calling it.

Basically for the reason that the following snippet works:
```lua
local M = {}

function M.something()
	print("something")
end

M.something()
```

and this one does not:
```lua
local M = {}

M.something()

function M.something()
	print("something")
end
```

the marked line nr.11 slipped into this. It would not be necessary to move it.